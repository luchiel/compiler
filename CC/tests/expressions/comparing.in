"a" > a || a == 'a',
b >= c && 0xA || 1.0 / 5 / 6