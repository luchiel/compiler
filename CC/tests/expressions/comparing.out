{,}
| 
+-{||}
| | 
| +-{>}
| | | 
| | +-{string a}
| | | 
| | +-{a}
| | 
| +-{==}
|   | 
|   +-{a}
|   | 
|   +-{char a}
| 
+-{||}
  | 
  +-{&&}
  | | 
  | +-{>=}
  | | | 
  | | +-{b}
  | | | 
  | | +-{c}
  | | 
  | +-{int 10}
  | 
  +-{float 1}
